plugins/out_s3/s3.c

```diff
@@ -1073,6 +1073,18 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
         if (chunk) {
             s3_store_file_unlock(chunk);
             chunk->failures += 1;
+            char create_time_string[80];
```

Comment:
variables must always be defined at top of function: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md